predicators/ground_truth_models/spot_env/nsrts.py

```diff
@@ -113,18 +114,23 @@ def _pick_object_from_top_sampler(state: State, goal: Set[GroundAtom],
     target_obj = objs[1]
     # Special case: if we're running dry, the image won't be used.
     if CFG.spot_run_dry:
-        pixel = (0, 0)
+        params_tuple = (0, 0, 0.0, 0.0, 0.0, 0.0)
     else:
         # Select the coordinates of a pixel within the image so that
         # we grasp at that pixel!
         target_detection_id = get_detection_id_for_object(target_obj)
         rgbds = get_last_captured_images()
         _, artifacts = get_last_detected_objects()
         hand_camera = "hand_color_image"
-        pixel = get_grasp_pixel(rgbds, artifacts, target_detection_id,
-                                hand_camera, rng)
-
-    return np.array(pixel)
+        grasp_pixel = get_grasp_pixel(rgbds, artifacts, target_detection_id,
+                                      hand_camera, rng)
+        rot_quat_tuple = (0.0, 0.0, 0.0, 0.0)
+        if target_obj.name == "ball":
+            rot_quat = math_helpers.Quat.from_pitch(np.pi / 2)
+            rot_quat_tuple = (rot_quat.w, rot_quat.x, rot_quat.y, rot_quat.z)
+            params_tuple = grasp_pixel + rot_quat_tuple
```

Comment:
oops there's a bug here that I missed until running -- `params_tuple` not defined. surprised linting didn't catch. I'll push the fix.